receiver/etwreceiver/receiver_windows.go

```diff
@@ -121,6 +155,9 @@ func newEtwReceiver(_ context.Context, cfg *WindowsEtwConfig, consumer consumer.
 		logger:          settings.Logger,
 		logsUnmarshaler: plog.JSONUnmarshaler{},
 		session:         session,
+		eventChan:       make(chan *plog.Logs, cfg.EventBufferSize),
```

Comment:
If i followed the code through correctly this should be changed, as it could be 0`ed.

```suggestion
		eventChan:       make(chan *plog.Logs, eventBufferSize),
```